lib/galaxy_social.py

```diff
@@ -94,10 +94,15 @@ def parse_markdown_file(self, file_path):
 
         metadata["media"] = [media.lower() for media in metadata["media"]]
 
+        invalid_media = [
+            media
+            for media in metadata["media"]
+            if media not in self.plugins_config_dict
+        ]
+        errors += f"- Invalid media `{', '.join(invalid_media)}` in metadata. you can only use `{', '.join(self.plugins_config_dict.keys())}`.\n"
+
         for media in metadata["media"]:
-            if media not in self.plugins_config_dict:
-                errors += f"- Invalid media `{media}` in metadata. Consider check the spelling or enabling/adding it to `plugins.yml`.\n"
-            elif media not in self.plugins:
+            if media in self.plugins_config_dict and media not in self.plugins:
```

Comment:
```suggestion
            if media not in self.plugins:
```

Reply:
if it doesn't check for the `media in self.plugins_config_dict`, it will throw an error and won't comment on the text in Github.